src/test/java/api/requests/RequestsAPILoanHistoryTests.java

```diff
@@ -43,6 +43,41 @@ void creatingRecallRequestChangesTheOpenLoanForTheSameItem() {
       is("recallrequested"));
   }
 
+  @Test
+  void checkOutShouldNotRecallLoanIfRecallRequestExistsForAnotherItemOfTheSameInstanceIfTlrIsEnabled() {
+    configurationsFixture.enableTlrFeature();
+    final var items = itemsFixture.createMultipleItemsForTheSameInstance(2);
+    final var pickupServicePointId = servicePointsFixture.cd1().getId();
+    var steve = usersFixture.steve();
+    var charlotte = usersFixture.charlotte();
+
+    ItemResource firstItem = items.get(0);
+    checkOutFixture.checkOutByBarcode(firstItem, steve);
+    requestsClient.create(new RequestBuilder()
+      .recall()
+      .forItem(firstItem)
+      .withPickupServicePointId(pickupServicePointId)
+      .by(charlotte));
```

Comment:
Can we use `RequestsFixture#recallItem` instead?

Reply:
Refactored